crates/holochain_sqlite/src/lib.rs

```diff
@@ -16,7 +16,7 @@
 //!
 //! Unpack and run the build commands per the README.md:
 //!
-//! ```
+//! ```text
```

Comment:
`text` not `sh`?

Reply:
haha, totally. Done: https://github.com/holochain/holochain/pull/4198/commits/dcd41c77a39af411416bc36731508a37f529476c